Tests/SessionManagerTests/SessionManagementTest.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 
 final class SessionManagementTest: XCTestCase {
 
-    var sessionID: String = "02666153283f10e9323e8e296032534d54314bc3296f2a3c47c9130346cc0dab"
+    var sessionID: String = "4b00111d33a3b8e4fbe29fcb8e4493e3985dda96cf618eee68aebc0ad9309e1c"
```

Comment:
Is there a js compatibility test?

Reply:
This test is for testing encrypt and decrypt function.